ci/integration.yml

```diff
@@ -0,0 +1,19 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/cf-acceptance-tests
+    tag: 895cf6752c8ec64af05a3a735186b90acd3db65a
+inputs:
+  - name: pr
+run:
+  path: sh
+  args:
+    - -e
+    - -c
+    - |
+      mkdir -p "${GOPATH}/src/github.com/alphagov/paas-rds-broker"
+      rsync -az pr/ "${GOPATH}/src/github.com/alphagov/paas-rds-broker/"
+      cd "${GOPATH}/src/github.com/alphagov/paas-rds-broker"
+      make integration
```

Comment:
Is there value in doing so, when Travis also runs them and submits a build status to the PR?